mlflow/server/js/src/experiment-tracking/components/experiment-page/utils/experimentPage.column-utils.ts

```diff
@@ -389,6 +393,15 @@ export const useRunsColumnDefinitions = ({
       initialHide: true,
     });
 
+    columns.push({
+      headerName: ATTRIBUTE_COLUMN_LABELS.DESCRIPTION,
+      colId: TAGS_TO_COLUMNS_MAP[ATTRIBUTE_COLUMN_SORT_KEY.DESCRIPTION],
+      field: 'tags',
+      cellRenderer: 'RunDescriptionCellRenderer',
+      initialWidth: 300,
+      initialHide: true,
```

Comment:
I think we can easily make this field sortable, backend supports this
```suggestion
      initialHide: true,
      sortable: true,
      headerComponentParams: {
        canonicalSortKey: ATTRIBUTE_COLUMN_SORT_KEY.DESCRIPTION,
      },
      cellClassRules: {
        'is-ordered-by': cellClassIsOrderedBy,
      },
```

Reply:
thanks, I accept this.